src/sysc/tracing/sc_vcd_trace.cpp

```diff
@@ -1374,9 +1374,8 @@ vcd_trace_file::cycle(bool this_is_a_delta_cycle)
 
     // Now do the actual printing
     bool time_printed = false;
-    vcd_trace* const* const l_traces = &traces[0];
-    for (int i = 0; i < (int)traces.size(); i++) {
-        vcd_trace* t = l_traces[i];
+    for (size_t i = 0; i < traces.size(); i++) {
```

Comment:
When touching these loops, why not use a range-based syntax instead and save the second line?
```suggestion
    for (vcd_trace * t : traces) {
```